lms/djangoapps/courseware/module_render.py

```diff
@@ -642,15 +642,15 @@ def rebind_noauth_module_to_user(module, real_user):
         field_data_cache_real_user = FieldDataCache.cache_for_descriptor_descendents(
             course_id,
             real_user,
-            module.descriptor,
```

Comment:
This method is called by two XBlocks. The one being converted and `LtiConsumerXBlock`. The later is an XBlock but this method has been working for it due to [this](https://github.com/edx/xblock-lti-consumer/blob/9ac5fda45c352fc456ab8b432bb967faf1a4661a/lti_consumer/lti_xblock.py#L609).

We can apply the same fix here too instead but I cannot imagine in which cases the current approach will not work?

Reply:
If no other remaining XModules use this function, then this change makes sense to me :+1: